certora/conf/GuardianStorage.conf

```diff
@@ -0,0 +1,26 @@
+{
+    "files": [
+        "contracts/modules/social_recovery/storage/GuardianStorage.sol",
+        "contracts/modules/social_recovery/SocialRecoveryModule.sol",
+        "certora/harnesses/SafeHarness.sol"
+    ],
+    "msg": "GuardianManager: General Ruleset",
+    "rule_sanity": "basic",
+    "loop_iter": "3",
+    "optimistic_loop": true,
+    "hashing_length_bound": "352",
+    "optimistic_hashing": true,
+    "solc": "solc-0.8.20",
+    "coverage_info": "basic",
+    "prover_args": [
+        "-smt_groundQuantifiers true -depth 0"
+    ],
```

Comment:
Are all of these parameters needed or just copied from somewhere? If so, it would be nice to comment which ones are needed and why.

Reply:
yes. We need all of there params.

Reply:
Documenting the requirements for the configuration file would be nice (so we know why the parameters are there).